### DISCUSSION TITLE
As a Monkeytype user, I would like to update my password

### DISCUSSION THREAD

**Discussion by Strik3ria** [monkeytypegame/monkeytype#1552] [Ideas] [answered]
I am a regular user of Monkey Type and seem to not have an option to update my password. I feel that this ability is critical to the overall application from a secure user stand point.

**Comment by unrealapex** [comment, marked as answer]
I agree, it would be nice to update your password also account deletion.

**Reply by Miodec** [reply]
Added password update popup to the settings. Should be live soon

**Reply by Strik3ria** [reply]
Thanks Team!!

**Reply by ShizukoV** [reply]
:D
